ci/codecov.sh

```diff
@@ -30,7 +30,7 @@ set -eux
 coverage_action=${1:-'<unset>'}
 
 if [[ "$coverage_action" == "setup" ]]; then
-    if [ -z "${B2_CI_VERSION:-}" ]; then
+    if [[ ${B2_CI_VERSION:-0} -gt 0 ]]; then
```

Comment:
The check here is "if it's the old version". Therefore the conditional should be:

`if [[ ${B2_CI_VERSION:-0} -lt 1 ]]; `